### DISCUSSION TITLE
rename req.getUrl to req.getPath

### DISCUSSION THREAD

**Discussion by hst-m** [uNetworking/uWebSockets.js#432] [Ideas]
Url is the full location string: protocol/host/path/query. req.getUrl should be called getPath

**Comment by pavulon** [comment]
If we're suggesting API changes, I think it would also be nice if compression and listen options were "actual" enums (`class enum` in c++ terms). Not sure how idiomatic would it be for the guys on the c++ side, but here I'd say it's a bit unusual to have them as just a bunch of exported constants.
